### PR TITLE
Create OuiSimplifiedBreadcrumbs and update OuiHeaderBreadcrumbs

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description

Create OuiSimplifiedBreadcrumbs and update OuiHeaderBreadcrumbs
image

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/components/header/header_breadcrumbs/header_breadcrumbs.tsx [hunk 1]
```diff
@@ -31,16 +31,32 @@
...
-  ...rest
-}) => {
-  const classes = classNames('ouiHeaderBreadcrumbs', className);
+export const OuiHeaderBreadcrumbs: FunctionComponent<
+  OuiBreadcrumbsProps & { simplify?: boolean }
+> = ({ className, breadcrumbs, simplify, ...rest }) => {
+  const classes = classNames(
+    'ouiHeaderBreadcrumbs',
+    {
+      'ouiHeaderBreadcrumbs--simplified': simplify,
```
Comment (virajsanghvi) [inline src/components/header/header_breadcrumbs/header_breadcrumbs.tsx:46]: is this needed here or should this just be in the component itself?

File: src/components/breadcrumbs/_breadcrumbs_simplified.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  max-width: $ouiBreadcrumbTruncateWidth;
+}
+
+.ouiBreadcrumbSeparator {
+  flex-shrink: 0;
+  display: inline-block;
+  margin-right: $ouiBreadcrumbSpacing;
+  width: 2px;
+  height: $ouiSize;
+  transform: translateY(-1px) rotate(15deg);
```
Comment (virajsanghvi) [inline src/components/breadcrumbs/_breadcrumbs_simplified.scss:77]: Is this a pre-existing implementation of the slashes?

Comment (zhongnansu) [reply src/components/breadcrumbs/_breadcrumbs_simplified.scss:77]: yes, taken from the code in oui 1.1

**Comment by virajsanghvi** [comment]
Can you add a changelog entry?

**Comment by zhongnansu** [comment]
> Can you add a changelog entry?

added

**Review by virajsanghvi — APPROVED** [review]

**Pull request merged**